cmd/cosmos-sdk-cli/template/app/app.go

```diff
@@ -0,0 +1,144 @@
+package app
```

Comment:
Seeing as the template should technically be a functioning project I'd propose that we remove it entirely and attempt to find/replace from the `examples/basecoin` project which is currently effectively supposed to act like the template at the moment... one less duplicate thing to maintain

Reply:
Is there anything unique about the current template project over the current basecoin example?

Reply:
I will check basecoin template and adapt it if it is suitable. For the time being, I just focussed on having a working setup for this tool - having the right build structure for the tool and producing a working version of the tool that can create a project that could be built and run out of the box. There is nothing special about the current templates.

Reply:
Cool thanks for pointing at basecoin